generators/client/templates/src/main/webapp/app/blocks/config/_compile.config.js

```diff
@@ -5,13 +5,15 @@
         .module('<%=angularAppName%>')
         .config(compileServiceConfig);
 
-    compileServiceConfig.$inject = ['$compileProvider', 'ENV'];
+    compileServiceConfig.$inject = ['$compileProvider'];
 
-    function compileServiceConfig($compileProvider, ENV) {
+    function compileServiceConfig($compileProvider) {
         // disable debug data on prod profile to improve performance
-        if(ENV === 'prod'){
-            $compileProvider.debugInfoEnabled(false);
-        }
+        
+        // TODO: Howto fetch the dynamic profile in this config.... can this be done elsewhere ?
```

Comment:
Exactly, that was one of the reasons why I wanted to remove that ENV constant. It's only updated when building with the -Pprod, and there's no obvious way to reset it in a CI scenario.

I would even rename that constant to clearly indicate that it will only be used for the debugInfo.

I don't like the idea of hardcoding an ENV into the artifact. We should be able to do everything with spring profiles.